docs/theme/package.json

```
@@ -3,6 +3,9 @@
   "main": "js/theme.js",
   "version": "1.0.0",
   "private": true,
+  "resolutions" : {
+    "**/elliptic": "6.5.3"
```

```suggestion
    "**/elliptic": "^6.5.3"
```

---

Doesn't `^6.5.3` mean semver-compatible with `6.5.3`? If so, that would include `6.5.2` as well. We probably need something like
```suggestion
    "**/elliptic": "^6.5.3 >=6.5.3"
```
Does that look right?

---

I don’t think so, `^` also adds a lower bound. See https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

---

Yeah, you're right. I stopped reading too early. Apologies.

---

@cocreature -- added the resolutions entry on package.json then reran yarn install so no manual edit of lock file

---

i'll add the range rather than specific version to both